src/version.ts

```diff
@@ -1,4 +1,4 @@
 // Project versions
 // Needs to be updated manually since in client-side, we don't have access to the package.json file.
-export const PLUGGY_JS_VERSION = '0.8.3'
-export const AXIOS_VERSION = '^0.21.1'
\ No newline at end of file
+export const PLUGGY_JS_VERSION = '0.9.3'
```

Comment:
the next major version increment should set the minor to 0
```suggestion
export const PLUGGY_JS_VERSION = '0.9.0'
```